ibis/expr/types/groupby.py

```diff
@@ -222,19 +223,43 @@ def _get_window(self):
                 order_by=self._window.order_by + self._order_by,
             )
 
-    def over(self, window) -> GroupedTable:
+    def over(
```

Comment:
Actually this method is not used anywhere, at least removing it doesn't break a single test.

Reply:
Yeah. I didn't realize we had support for applying a window over a group by. This makes sense to me though. It avoids having to duplicate a window if the rows/range specification are more complex than can be done with `.group_by(...).mutate(...)`